Automattic/newspack-blocks
# Code style (JS): prettier and eslint configuration

### All Submissions:






### Changes proposed in this Pull Request:



#326 fixes linting errors for JS files. This PR: 
- configures `prettier` to operate without conflicts with `eslint`
- formats the JS files so they adhere to our prettier configuration

Note that this is a draft, as it should be merged after #326 is merged.

### How to test the changes in this Pull Request:

1. run `npm run lint:js` - there should be no errors
2. run `npm run format:js` - there should be no diff (code is already formatted in this PR)

### Other information:
